core/backend/package.json

```diff
@@ -97,6 +97,22 @@
     "plugins": [
       "@itwin"
     ],
-    "extends": "plugin:@itwin/itwinjs-recommended"
+    "extends": "plugin:@itwin/itwinjs-recommended",
+    "rules": {
+      "@typescript-eslint/no-non-null-assertion": "error"
+    },
+    "overrides": [
+      {
+        "files": [
+          "*.test.ts",
+          "*.test.tsx",
```

Comment:
No need to specify any `.tsx` file extensions for the backend code.

Reply:
Fixed, thanks